app/assets/javascripts/header.js

```diff
@@ -94,4 +68,36 @@ function setHeight() {
   $('#header_space').css({"height": the_height});
 }
 
+function expandButton(event) {
+  hideSmallTopNav();
+  console.log(event.data);
```

Comment:
do we want to keep JS console logging in at this point? I presume this was useful when debugging!